### PR TITLE
'.coafile' was not found now occurs only once

### PR DESCRIPTION
**Pull request by shreyateeza**
coala-quickstart --ci is now emitting warning "The default coafile '.coafile' was not found." only once.

Closes https://github.com/coala/coala-quickstart/issues/62

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: coalib/settings/ConfigurationGathering.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import logging
 
 from coalib.collecting.Collectors import (
-    collect_all_bears_from_sections, filter_section_bears_by_languages)
+                                collect_all_bears_from_sections, filter_section_bears_by_languages)  # Ignore PEP8Bear
```
Comment (nalinbhardwaj) [inline coalib/settings/ConfigurationGathering.py]: Unrelated change.

Comment (Mixih) [reply coalib/settings/ConfigurationGathering.py]: Split into a separate commit if you still want to add this change in.

File: coalib/settings/.coafile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[cli]
+bears = PEP8Bear,PyUnusedCodeBear
```
Comment (Mixih) [inline coalib/settings/.coafile]: I'm not sure if we want to ship this here, or even as a part of the main coala install. maybe you should modify `coala-quickstart` instead. Why are you adding this file?

File: coalib/settings/ConfigurationGathering.py [hunk 3]
```diff
@@ -266,7 +267,7 @@ def load_configuration(arg_list,
             save = True
 
         coafile_sections = load_config_file(config,
-                                            silent=save or silent)
+                                            silent=save)
```
Comment (Mixih) [inline coalib/settings/ConfigurationGathering.py]: Just force silent to true as per https://github.com/coala/coala-quickstart/issues/62#issuecomment-289267092. this may break tests though which may need modification.

File: coalib/settings/ConfigurationGathering.py [hunk 4]
```diff
@@ -252,7 +252,8 @@ def load_configuration(arg_list,
         user_sections = load_config_file(
             Constants.user_coafile, silent=True)
 
-        default_config = str(base_sections['default'].get('config', '.coafile'))
+        default_config = str(
```
Comment (Mixih) [inline coalib/settings/ConfigurationGathering.py]: unrelated change

File: coalib/settings/ConfigurationGathering.py [hunk 5]
```diff
@@ -105,7 +105,7 @@ def merge_section_dicts(lower, higher):
     return lower
 
 
-def load_config_file(filename, log_printer=None, silent=False):
+def load_config_file(filename, log_printer=None, silent=True):
```
Comment (jayvdb) [inline coalib/settings/ConfigurationGathering.py]: changing the default for this arg is not within the scope of your issue.
